index.html

```diff
@@ -8,6 +8,8 @@
 </head>
 <body>
     <ul>
+        <thead>Lista de Cursos de DEVOPS da Alura</thead>
```

Comment:
BobMad pra mim você é o melhor .... deste minuto ...rs

Reply:
To sabendo